### PR TITLE
 Use relative URLs to refer to static resources

### PR DESCRIPTION
**Pull request by stefwalter**
This applies when logged in. The static resources are available under /cockpit/static

Depends on #1909

### CODE REVIEW

**Comment by stefwalter** [comment]
Rebased

**Comment by stefwalter** [comment]
Testing failed:

```
error: Domain snapshot not found: no domain snapshot with matching name 'initialized'
```

**Comment by stefwalter** [comment]
Rebased

File: src/static/login.html [hunk 1]
```diff
@@ -527,13 +527,13 @@
   font-family: 'Open Sans';
   font-style: normal;
   font-weight: 400;
-  src: url('/static/fonts/OpenSans-Regular-webfont.woff') format('woff');
+  src: url('/cockpit/static/fonts/OpenSans-Regular-webfont.woff') format('woff');
```
Comment (mvollmer) [inline src/static/login.html]: Shouldn't this also be relative?
```
src: url('./cockpit/static/fonts/OpenSans-Regular-webfont.woff') format('woff');
```

File: src/static/login.html [hunk 2]
```diff
@@ -527,13 +527,13 @@
...
 }
 @font-face {
   font-family: 'Open Sans';
   font-style: normal;
   font-weight: 700;
-  src: url('/static/fonts/OpenSans-Bold-webfont.woff') format('woff');
+  src: url('/cockpit/static/fonts/OpenSans-Bold-webfont.woff') format('woff');
```
Comment (mvollmer) [inline src/static/login.html]: Relative?

File: src/static/login.html [hunk 3]
```diff
@@ -651,7 +651,7 @@
 
         /* Login page specific overrides */
         body {
-            background: url("/static/bg-login.jpg") no-repeat 50% 0;
+            background: url("/cockpit/static/bg-login.jpg") no-repeat 50% 0;
```
Comment (mvollmer) [inline src/static/login.html]: Relative?

File: src/static/login.html [hunk 4]
```diff
@@ -686,13 +686,13 @@
   </head>
   <body class="login-pf">
     <span id="badge">
-      <img src="/static/logo.png" alt="" />
+      <img src="/cockpit/static/logo.png" alt="" />
```
Comment (mvollmer) [inline src/static/login.html]: Relative?

File: src/static/login.html [hunk 5]
```diff
@@ -686,13 +686,13 @@
...
     </span>
     <div class="container">
       <div class="row">
         <div class="col-sm-12">
           <div id="brand">
-            <img src="/static/brand-large.png" alt="Branding Large">
+            <img src="/cockpit/static/brand-large.png" alt="Branding Large">
```
Comment (mvollmer) [inline src/static/login.html]: Relative?

File: src/static/login.html (hunk 1 above)
Comment (stefwalter) [inline src/static/login.html]: The login page can't be relative yet. It is also served at other paths than '/'

**Pull request closed without merging**